http_server/queries/messages_reported/messages_reported_check_existing.php

```diff
@@ -16,7 +16,7 @@ function messages_reported_check_existing($pdo, $message_id)
     
     $count = $stmt->fetchColumn();
     
-    if ($count == 0) {
+    if ((int) $count == 0 || $count == false || empty($count)) {
```

Comment:
It seem like `return (boolean) $stmt->fetchColumn();` would take care of all of the possibilities here.

Also, cool trick with the fetchColumn. Didn't know that one existed :)

Reply:
Aha! Neat. Learn something new every day, I guess. :)